lib/active_model_serializers/adapter/json.rb

```diff
@@ -9,6 +9,14 @@ def serializable_hash(options = nil)
         self.class.transform_key_casing!(serialized_hash, instance_options)
       end
 
+      def root
+        if instance_options.key?(:root)
+          instance_options[:root]
+        elsif serializer.json_key
+          serializer.json_key.to_sym
```

Comment:
Not sure why `.to_sym` is called here, just transposed [this](https://github.com/rails-api/active_model_serializers/pull/1782/files#diff-426cb6b4484fa95a18649ea3cc74474aL79).